.gitignore

```diff
@@ -1,6 +1,9 @@
 *build*
+!.ci/**
+!.github/**
+!.gitignore
```

Comment:
Why do you need this? I mean you don't want to create a whitelist here, or?

Reply:
The inclusion (!) was needed because the build exclusion was blocking the `build-kit` dir and the `build_and_test.sh` file

Reply:
Oh, ok. Then better fix this, instead. I.e. `build*/` would work.